homestead.root/usr/share/clearwater/infrastructure/scripts/homestead

```diff
@@ -36,6 +36,8 @@
 
 hss_port=3868
 hs_secure_listen_port=5658
+acl_required=false
+check_destination_host=true
```

Comment:
What's our plan for changing the default value of `check_destination_host` in future installs?